apps/omg_watcher/lib/web/controllers/transaction.ex

```diff
@@ -92,6 +128,35 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
             spender1: "92EAD0DB732692FF887268DA965C311AC2C9005B",
             spender2: "92EAD0DB732692FF887268DA965C311AC2C9005B"
           })
+        end,
+      Output:
+        swagger_schema do
+          title("The Output")
+
+          properties do
+            amount(:integer, "Amount of the currency spent in inputs", required: true)
+            owner(:string, "Address of output's owner", required: true)
+          end
+
+          example(%{
+            "amount" => 97,
+            "owner" => "B3256026863EB6AE5B06FA396AB09069784EA8EA"
+          })
+        end,
+      Outputs:
+        swagger_schema do
+          title("The array of outputs")
+          type(:array)
+          items(Schema.ref(:Output))
+        end,
+      Post_Transaction_Body:
+        swagger_schema do
+          title("The POST method '/transaction' request body schema")
```

Comment:
I think this title is redundant. We'll have that '/transaction' in path. I think  "Inputs and outputs for transaction" would do.